### PR TITLE
Avatar component with badge has created

### PR DESCRIPTION
**Pull request by balaDharmaraj-81**
Currently creating only components. The documentation and demo website will create after completing the components.

### CODE REVIEW

File: css/avatar.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    font-size: 4rem;
+}
+
+.neog__avatar.lg{
+    width: 6rem;
+    height: 6rem;
+    font-size: 3rem;
+}
+
+.neog__avatar.md{
```
Comment (arulk8) [inline css/avatar.css:37]: instead .neog__avatar.md you can directly use .md(But it depends on your design)

Comment (arulk8) [reply css/avatar.css:37]: UI looks good to me

Comment (balaDharmaraj-81) [reply css/avatar.css:37]: Thank you so much Arul ji for the review/comments. I thought, in a whole application perspective ".md" is a global class name and this class name can be used for any component for different responsive scenarios. So to make it personalized for avatar, just grouped the class names.

**Review by raieegulhane — COMMENTED** [review]
Reviewed this. Looks good to me. You can use more descriptive class names. That way it will be easier for anyone to understand your code.

**Pull request merged**